### PR TITLE
Changed copy on email signup error

### PR DESCRIPTION
**Pull request by jameshupp**
## Changes

Adjusted copy to reflect:
1. The signup functions.
2. Signup errors are most likely to be about communicating with the vendor, so sending them to another signup is not going to be super helpful.

## Review
@jimmynotjim 
@sarahsimpson09 

## Note
The last line might be a little odd. I just thought it might be a way to get more business units excited about sending emails. Regular communication is good for us!

### CODE REVIEW

File: govdelivery-subscribe/error/index.html [hunk 1]
```diff
@@ -11,13 +11,13 @@
...
-        this beta release. It will be available soon. For now:
+        We couldn’t complete your signup request. Try<br>
+        again in a little while. For now:
     </p>
     <ol class="short-desc">
         <li>Keep exploring <a href="http://beta.consumerfinance.gov">beta.consumerfinance.gov</a>.</li>
-        <li>Check back often to follow our progress.</li>
-        <li>Sign up for general CFPB updates at <a href="http://www.consumerfinance.gov">www.consumerfinance.gov</a>.</li>
+        <li>Check back often to follow our progress and find new sources of information.</li>
+        <li><a href="/contact-us/"></a>Let us know</a> if there’s a particular line of work you want to hear about regularly.</li>
```
Comment (anselmbradford) [inline govdelivery-subscribe/error/index.html]: Extra closing tag pushes text outside of the link:
`<a href="/contact-us/"></a>Let us know</a>` > `<a href="/contact-us/">Let us know</a>`

File: govdelivery-subscribe/error/index.html [hunk 2]
```diff
@@ -11,13 +11,13 @@
 {% block content_main %}
     <h1><span class="cf-icon cf-icon-error-round"></span> Not just yet!</h1>
     <p class="short-desc">
-        We’re still building the email signup function for <br>
-        this beta release. It will be available soon. For now:
+        We couldn’t complete your signup request. Try<br>
```
Comment (jimmynotjim) [inline govdelivery-subscribe/error/index.html]: While extremely fringe, I realized today this error page will also be displayed if the user doesn't enter an email address. Do we want to point out that possibility here?

Comment (anselmbradford) [inline govdelivery-subscribe/error/index.html]: Is disabling the Sign up button till a user has entered an email address an option? Is there any reason to allow a user to click that button and go to an error page?

Comment (jameshupp) [inline govdelivery-subscribe/error/index.html]: I think I agree with Ans on this. It just seems unnecessary to put someone through the pain of leaving the signup page in the first place.

Though another question would be: do we validate that it's structured as an email address? If we do, then it seems even easier to not let them submit in the first place. I don't know if GovDelivery validates—a user has to click to verify, so they may not care about the submission structure—but if they do we might have to serve the error then, right?

Comment (jimmynotjim) [inline govdelivery-subscribe/error/index.html]: Good points. I'll add that to the list of things to tackle in the Ajax task.

**Comment by jimmynotjim** [comment]
:+1: Merge when ready.

File: govdelivery-subscribe/error/index.html [hunk 3]
```diff
@@ -9,15 +9,16 @@
 {%- endblock %}
 
 {% block content_main %}
-    <h1><span class="cf-icon cf-icon-error-round"></span> Not just yet!</h1>
+    <h1><span class="cf-icon cf-icon-error-round"></span> That didn't work.</h1>
```
Comment (anselmbradford) [inline govdelivery-subscribe/error/index.html]: Minor:
Change `didn't` to `didn’t`.

File: govdelivery-subscribe/error/index.html [hunk 4]
```diff
@@ -9,15 +9,16 @@
...
     <p class="short-desc">
-        We’re still building the email signup function for <br>
-        this beta release. It will be available soon. For now:
+        We couldn’t complete your signup request. Try <br>
+        to submit your address again in a little while. For <br>
```
Comment (anselmbradford) [inline govdelivery-subscribe/error/index.html]: Should the line breaks be at the end of the sentences? Like: 

```html
We couldn’t complete your signup request. <br>
Try to submit your address again in a little while. <br>
For now:
```

Or is that existing line breaks intentional?

Comment (jameshupp) [inline govdelivery-subscribe/error/index.html]: I hate all the line breaks, but the way the text was laid out originally, they were supposed to stay short enough that the text doesn't break differently on mobile or desktop. I've just copied those line lengths.

I'd be happy to have a new container created for the copy that keeps it that way, but this seemed easy enough for now. It's too short a paragraph to stretch across the page; looked kind of funny.

Comment (anselmbradford) [inline govdelivery-subscribe/error/index.html]: :+1:

Comment (sarahsimpson09) [inline govdelivery-subscribe/error/index.html]: This looks good to me!

**Pull request merged**